### DISCUSSION TITLE
[Weekly Review Report]: June 17 - 21

### DISCUSSION THREAD

**Discussion by susancoort** [wikipathways/wikipathways-help#123] [curation-reports]
### Recent changes

 * WP5485: New pathways -> added the approved tag
 * Updated two AOP tags

### Under construction

 * All under construction tags are kept.

### New pathways

 * WP5485: Post-Covid Neuroinflammation

### Deleted Pathways

 * none

### Update Approved pathway tag

- [X] All Approved pathways tags were updated!

**Comment by khanspers** [comment]
Thank you @susancoort !
